src/meson.build

```diff
@@ -0,0 +1,85 @@
+gnotime_cfg_data = configuration_data()
+gnotime_cfg_data.set_quoted('DATADIR', get_option('prefix') / get_option('datadir') / 'gnotime')
+gnotime_cfg_data.set_quoted('GETTEXT_PACKAGE', 'gnotime-2.0')
+gnotime_cfg_data.set_quoted('GNOME_ICONDIR', 'NONE' / get_option('datadir') / 'pixmaps')
+gnotime_cfg_data.set_quoted('GNOMELOCALEDIR', get_option('prefix') / get_option('localedir'))
+gnotime_cfg_data.set_quoted('GTTDATADIR', get_option('prefix') / get_option('datadir') / 'gnotime')
+gnotime_cfg_data.set_quoted('GTTGLADEDIR', get_option('prefix') / get_option('datadir') / 'gnotime')
+gnotime_cfg_data.set_quoted('LIBDIR', get_option('prefix') / get_option('libdir'))
```

Comment:
This macro is never used, as far as I can tell.

Reply:
See above